numalogic/synthetic/anomalies.py

```diff
@@ -51,6 +67,13 @@ def __init__(self, ref_df: pd.DataFrame, anomaly_type="global", anomaly_ratio=0.
     def injected_cols(self) -> List[str]:
         return self.__injected_cols
 
+    def add_impact_sign(self) -> int:
+        if self.anomaly_sign == "positive":
+            return 1
+        if self.anomaly_sign == "negative":
+            return -1
+        raise AttributeError(f"Invalid anomaly sign provided: {self.anomaly_sign}")
```

Comment:
This will of type ValueError. Read https://docs.python.org/3/library/exceptions.html#AttributeError
```suggestion
        raise ValueError(f"Invalid anomaly sign provided: {self.anomaly_sign}")
```